recipes/folly/all/conan_deps.cmake

```
@@ -0,0 +1,47 @@
+cmake_minimum_required(VERSION 3.17)
+# Set the dependency flags expected by https://github.com/facebook/folly/blob/v2023.12.18.00/CMake/folly-deps.cmake
+
+macro(custom_find_package package_name variable_prefix)
+    find_package(${package_name} REQUIRED CONFIG ${ARGN}
+        # Allow only Conan packages
+    )
```

```suggestion
    find_package(${package_name} REQUIRED CONFIG ${ARGN})
```
Should remove the leftover comment as well.